vendor/assets/javascripts/fluxxor.min.js

```diff
@@ -0,0 +1,2 @@
+//# sourceMappingURL=fluxxor.min.js.map
```

Comment:
Can we make a policy not to include minified javascripts in our repo? Rails asset pipeline minifies it for us in production, and in development it's nice to be able to follow stack traces into libraries. they also are a pain when I grep in the whole repo. finally, the annotated source is ideal to have in the repo as a reference.